pages/browser-support.html

```diff
@@ -4,37 +4,32 @@
 
 <h2>Current Active Support</h2>
 
-<table id="support-matrix">
-	<thead>
-		<tr>
-			<th></th>
-			<th>Internet Explorer</th>
-			<th>Edge, Opera, Yandex.Browser</th>
-			<th>Chrome, Firefox, Safari</th>
-			<th>iOS</th>
-			<th>Android</th>
-		</tr>
-	</thead>
-	<tbody>
-		<tr>
-			<th>jQuery</th>
-			<td>9+</td>
-			<td rowspan="2">Current</td>
-			<td rowspan="2">(Current - 1) and Current</td>
-			<td rowspan="2">7+</td>
-			<td rowspan="2">2.3, 4.0+</td>
-		</tr>
-		<tr>
-			<th>jQuery Compat</th>
-			<td>8+</td>
-		</tr>
-	</tbody>
-</table>
+<h3>Desktop</h3>
+<ul>
+	<li>Chrome: (Current - 1) and Current</li>
+	<li>Edge: Current</li>
+	<li>Firefox: (Current - 1) and Current</li>
+	<li>Internet Explorer: 9+</li>
+	<li>Safari: (Current - 1) and Current</li>
+	<li>Opera: Current</li>
```

Comment:
I did it on purpose. Opera is a secondary browser (same as Yandex.Browser), I wanted to mention all primary ones first.